### ISSUE TITLE
Write a small tutorial about how to set and use "Browser Link" with ASP.NET Core

### ISSUE THREAD

**Issue by ncarandini** [aspnet/Docs#2258]
Hi, I'd like to know if I can contribute writing a small tutorial about how to use Browser Link with ASP.NET Core.

Something like the [Using Browser Link in Visual Studio 2013](https://www.asp.net/visual-studio/overview/2013/using-browser-link) post written by Mike Wasson, with a preamble for the things needed to set it for an ASP.NET Core project:

1. Add the `Microsoft.VisualStudio.Web.BrowserLink.Loader` package with one of these options:
    - Manually editing project.json for .NET Core 1.0.x
    - Manually editing .csproj for .NET Core 1.1.0
    - Using NuGet package Manager
2. Add the `app.UseBrowserLink()` in the Startup.cs file:
```
if (env.IsDevelopment())
{
    app.UseBrowserLink();
}
```
Let me know if it can be helpful. I know it's a small contribution... :-)

**Comment by Rick-Anderson** [comment]
Go ahead. Add it to client side dev.

**Comment by ncarandini** [comment]
@Rick-Anderson I wrote the draft, made the PR  #2445 and requested the review.

Because I've used the structure and some (revisited and updated when needed) text from the original [Using Browser Link in Visual Studio 2013](https://www.asp.net/visual-studio/overview/2013/using-browser-link) article written by Mike Wasson, I've added a note at the end of the article to give him credit for his original work.

Maybe we can ask @MikeWasson if he likes / prefers to be in the authors list.
